roles/neutron-common/templates/etc/serverspec/neutron_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe file('/etc/neutron/rootwrap.d') do
+  it { should be_owned_by 'root' }
+  it { should be_grouped_into 'root' }
+  it { should be_writable.by('owner') }
+end    
+
+describe file('/opt/openstack/current/neutron/etc/neutron/rootwrap.d') do
+  it { should be_owned_by 'root' }
+  it { should be_grouped_into 'root' }
+  it { should be_writable.by('owner') }
+end  
```

Comment:
Checking both of these seems weird. The second one only matters because the first is a symlink to the second. But I don't think it really matters, except that any changes will cause a double error.